doc/source/whatsnew/v0.24.0.txt

```diff
@@ -14,6 +14,7 @@ Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 - :func:`to_datetime` now supports the ``%Z`` and ``%z`` directive when passed into ``format`` (:issue:`13486`)
 - :func:`to_csv` now supports ``compression`` keyword when a file handle is passed. (:issue:`21227`)
+- ``ExtensionArray`` has a ``ExtensionOpsMixin`` factory that allows default operators to be defined (:issue:`20659`, :issue:`19577`)
```

Comment:
make a separate sub-section. pls expand this a bit, I know what you mean, but I doubt the average reader does.

Reply:
Is the note here still accurate or has the factory been changed to just mixins?

Reply:
I've written a section for whatsnew